src/Tests.Microsoft.Security.Utilities.Core/IdentifiableSecretsTests.cs

```diff
@@ -521,9 +535,12 @@ public void IdentifiableSecrets_TryValidateCommonAnnotatedKey_RejectInvalidSigna
                                                                              new byte[9],
                                                                              new byte[3]);
 
-            bool result = IdentifiableSecrets.TryValidateCommonAnnotatedKey(validKey, validSignature);
+            bool result = TryValidateCommonAnnotatedKeyHelper(validKey, validSignature);
             result.Should().BeTrue(because: "a generated key should validate");
 
+            result = TryValidateCommonAnnotatedKeyHelper(validKey, base64EncodedSignature: "aBcD");
+            result.Should().BeFalse(because: "although the signature comparison is case-insentitive, the signature argument must have consistent case.");
```

Comment:
```suggestion
            result.Should().BeFalse(because: "although the signature comparison is case-insentitive, the signature argument must have consistent case");
```